### PR TITLE
Distribution package for apicurio Connect converter

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: distro/connect-converter/pom.xml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        <version.geronimo.config>1.2.2</version.geronimo.config>
+        <distro.version.kafka>2.4.0</distro.version.kafka>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.kafka</groupId>
+                <artifactId>connect-json</artifactId>
+                <version>${distro.version.kafka}</version>
```
Comment (alesj) [inline distro/connect-converter/pom.xml]: Shouldn't this be inlined with what we have in top pom.xml ?

File: distro/connect-converter/pom.xml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
+            <plugin>
```
Comment (alesj) [inline distro/connect-converter/pom.xml]: Is this really needed?
Do we run this plugin all the time?

File: pom.xml [hunk 3]
```diff
@@ -146,6 +146,7 @@
 		<version.checkstyle.plugin>3.1.0</version.checkstyle.plugin>
         <version.properties.plugin>1.0.0</version.properties.plugin>
         <version.dependency.plugin>3.1.1</version.dependency.plugin>
+        <version.assembly.plugin>3.1.1</version.assembly.plugin>
```
Comment (alesj) [inline pom.xml]: Didn't we use assembly plugin before?

File: distro/connect-converter/pom.xml (hunk 1 above)
Comment (jpechane) [reply distro/connect-converter/pom.xml]: @alesj No, this is intentional as this depends on to what Kafka Connect version you want to deploy. Moreover you have desynced Kakfa and Kafka Connect. Kafka clients are brought from Quarkus in version 2.3.x and you are using Connect in version 2.2.x
It is not possible to align both on the same version -even if it is desirable as you are using Connect 2.2.x specific API and upgrading breaks the build.

Comment (alesj) [reply distro/connect-converter/pom.xml]: @jpechane OK, I see ... 
Yeah, we depend on Quarkus ... so we can move the Connect to the same 2.3.x ?

File: pom.xml (hunk 3 above)
Comment (jpechane) [reply pom.xml]: It is in version`2.2-beta-5` and the version is defined out of the scope of the project. I think this should be definel locally.

File: distro/connect-converter/pom.xml (hunk 1 above)
Comment (jpechane) [reply distro/connect-converter/pom.xml]: @alesj You can but the code needs to be updated then - the file `io/apicurio/registry/connector/ConnectorApplication.java`.

Just a sidenote, maybe you can look at `io.debezium.embedded.EmbeddedEngine` there might be a place for cooperation

File: distro/connect-converter/pom.xml (hunk 2 above)
Comment (jpechane) [reply distro/connect-converter/pom.xml]: Right now it is run whener the main project is built.

**Pull request merged**